tox.ini

```diff
@@ -11,11 +11,15 @@ deps =
 commands = py.test
 
 [pytest]
-addopts=--cov=uswds_forms --cov-report=term --cov-report=html
+addopts=--cov=uswds_forms --cov-report=term --cov-report=html --ignore=venv
```

Comment:
Ah yes, good call. I do virtualenvs differently so I didn't see this, but it's a good idea.